src/_tutorials/web/get-started.md

```diff
@@ -73,10 +71,8 @@ For a list of available IDEs, see the
 To create a web app from the command line, use these commands:
 
 ```terminal
-$ mkdir quickstart
+$ dart create -t web-simple quickstart
 $ cd quickstart
```

Comment:
Let's move this down to the "Run the app" section (in the same code block as `webdev serve`).

Reply:
Sure

Reply:
Sure, done!